docs/fast_data/configuration/projection_storer.md

```diff
@@ -360,37 +325,6 @@ in the type expected by the data model for that specific field on which the cast
 
 Below is provided an example of cast functions implementation, one for each supported programming language.
 
-<details><summary>Custom Cast Function (Kotlin - mapToAddressType.kts)</summary>
-<p>
-
-```kotlin
-package castFunctions
-
-val addressMapping = mapOf(
-  1 to "SHIPPING",
-  2 to "BILLING",
-  3 to "LIVING",
-)
-
-// NOTE: the name of the function must correspond to
-//       the key associated to the file containing it
-fun mapToAddressType(value: Any, fieldName: String): String? {
```

Comment:
Would it be possible to convert this custom cast function in Javascript, instead of removing it?
The secondary goal of this function was to show that it was also possible not only to change the type, but also the value itself (e.g. mapping it against a dictionary)

Reply:
@danibix95 sure, I reverted the function back into the code and changed it according to js